### PR TITLE
Tool unit tests

### PR DESCRIPTION
**Pull request by cszsol**

### CODE REVIEW

**Comment by cszsol** [comment]
This PR has to be merged first
https://github.com/BlueBrain/neuroagent/pull/32

File: .github/workflows/changelog.yml
Comment (WonderPG) [inline .github/workflows/changelog.yml:1]: This file should be brought back

File: swarm_copy/tools/bluenaas_memodel_getall.py [hunk 1]
```diff
@@ -55,7 +57,7 @@ async def arun(self) -> PaginatedResponseUnionMEModelResponseSynaptomeModelRespo
         response = await self.metadata.httpx_client.get(
             url=f"{self.metadata.bluenaas_url}/neuron-model/{self.metadata.vlab_id}/{self.metadata.project_id}/me-models",
             params={
-                "simulation_type": self.input_schema.model_type,
```
Comment (WonderPG) [inline swarm_copy/tools/bluenaas_memodel_getall.py:58]: This is an ME model response, not a simulation one. `simulation_type` variable name is not very suitable in this case.

File: swarm_copy/tools/bluenaas_memodel_getone.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ class MEModelGetOneMetadata(BaseMetadata):
 class InputMEModelGetOne(BaseModel):
     """Inputs for the BlueNaaS single-neuron simulation."""
 
-    model_id: str = Field(
```
Comment (WonderPG) [inline swarm_copy/tools/bluenaas_memodel_getone.py:27]: Same here, `simulation_id` is not a fitting name.

File: swarm_copy_tests/tools/test_literature_search_tool.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                "article_title": "Article title",
+                "article_authors": ["Author1", "Author2"],
+                "paragraph": "This is the paragraph",
+                "section": "fake_section",
+                "article_doi": "fake_doi",
+                "journal_issn": "fake_journal_issn",
+            }
+            for _ in range(reranker_k)
+        ]
+
```
Comment (WonderPG) [inline swarm_copy_tests/tools/test_literature_search_tool.py:34]: Why not just use the `httpx_mock` fixture as in the other tests ?

**Review by WonderPG — COMMENTED** [review]
Couple of comments, thank you !

File: swarm_copy/tools/bluenaas_memodel_getall.py (hunk 1 above)
Comment (kanesoban) [reply swarm_copy/tools/bluenaas_memodel_getall.py:58]: I changed it to memodel_type, and at the other places as well.

File: swarm_copy/tools/bluenaas_memodel_getone.py (hunk 2 above)
Comment (kanesoban) [reply swarm_copy/tools/bluenaas_memodel_getone.py:27]: I changed it to memodel id

File: swarm_copy_tests/tools/test_literature_search_tool.py (hunk 3 above)
Comment (kanesoban) [reply swarm_copy_tests/tools/test_literature_search_tool.py:34]: fixed

**Comment by BoBer78** [comment]
There is one test missing in `test_get_morpho_tool` (the one testing the query). 
Appart from that there are a couple issues with the CI not passing because of the recent changes. After that it should be good to go !

**Review by BoBer78 — APPROVED** [review]

**Pull request merged**